src/content/admin/resource-manager.mdx

```diff
@@ -108,6 +90,18 @@ If you find that the Resource Manager's recommendations are not suitable for you
 Update the annotation `dev.okteto.com/resource-manager` to `false` to your pod's metadata to prevent automatic adjustments.
 Roll back to previous resource settings manually if needed.
 
+## Disabling the Resource Manager
+If you need to disable the Resource Manager temporarily, you can do so by:
+
+- Using the toggle switch in the Admin UI to disable the Resource Managers automatic adjustments
+<!-- - To uninstall Resource Manager, you can set the `resourceManager` helm setting to `enabled: false` -->
+
 ## Troubleshooting and Support
 ### Metric Server Availability
-The Resource Manager relies on the Kubernetes Metric Server. If the Metric Server is unavailable, the Okteto UI will display an error message. You can troubleshoot the Metric Server or refer to the documentation for further guidance.
\ No newline at end of file
+The Resource Manager relies on the Kubernetes Metric Server. If the Metric Server is unavailable, the Okteto UI will display an error message. You can troubleshoot the Metric Server or refer to the documentation for further guidance.
+
+### Known Limitation: Resource Manager Support for `ReplicaSet` and `CronJob`
```

Comment:
@pchico83 @andreafalzetti Is this too much detail to add here?

Reply:
I think the place is good, I would make the heading shorter and without backticks as it looks odd in the sidebar. WDYT?

<img width="297" alt="Screenshot 2024-11-06 at 19 53 21" src="https://github.com/user-attachments/assets/450bdcc2-f963-4d0f-aae1-8dba2c41b0cc">

```suggestion
### Known Limitation: Support for ReplicaSet and CronJob
```